projects/json-c/build.sh

```diff
@@ -20,11 +20,11 @@ cmake -DBUILD_SHARED_LIBS=OFF ..
 make -j$(nproc)
 cd ..
 
-cp $SRC/*.dict $OUT/
+cp fuzz/*.dict $OUT/
```

Comment:
What's the "fuzz/" directory, and why do we want to use it instead of $SRC?  I don't see it mentioned at eg https://github.com/google/oss-fuzz/blob/master/docs/getting-started/new_project_guide.md

Reply:
That is the [fuzz](https://github.com/json-c/json-c/tree/master/fuzz) directory inside the json-c repository. The build script is executed inside a checkout of json-c (see https://github.com/google/oss-fuzz/blob/master/projects/json-c/Dockerfile#L20). I can change this to `$SRC/json-c/fuzz` for a more explicit path.

Reply:
@hawicz Anything else I could add to clear up this PR?

If you want to try the changes out for yourself you can clone this branch via `git clone -b use-upstream-json-c-fuzz-test git@github.com:simonresch/oss-fuzz.git`. Then inside the `oss-fuzz` directory run `python3 infra/helper.py introspector json-c` to build all fuzz tests and briefly execute them. With the changes from this PR it should execute all fuzz tests from https://github.com/json-c/json-c/tree/master/fuzz instead of the single fuzz test that is currently configured in the oss-fuzz repository.

Reply:
lgtm